config/initializers/audit_events.rb

```diff
@@ -4,6 +4,6 @@
 # Put here all custom listener names, for later usage in Audit Instrumentation
 #
 # Example:
-# Audit::Subscribe.listen :admin, :quest_user
+# Audit::Subscribe.listen :internal, :quest_user
```

Comment:
Why are we changing from `admin` to `internal` here instead of the other way around?

Reply:
This is actually just fixing the comment so it aligns with the code! I did not rename the internal (pun intended) audit trail name